src/plugins/Wallet/real.ts

```diff
@@ -47,8 +48,37 @@ async function sendTx<R, T extends TransactionObject<R>>(txObject: T, tx: Tx = {
                     gas,
                     gasPrice,
                 })
+                /**
+                 * @returns {String}
```

Comment:
generally you don't need a JSDoc type annotation cause the type system do the work